src/install/npm.zig

```diff
@@ -1191,142 +1191,158 @@ pub const PackageManifest = struct {
 
                     inline for (dependency_groups) |pair| {
                         if (prop.value.?.asProperty(comptime pair.prop)) |versioned_deps| {
-                            const items = versioned_deps.expr.data.e_object.properties.slice();
-                            var count = items.len;
-
-                            var this_names = dependency_names[0..count];
-                            var this_versions = dependency_values[0..count];
+                            if (versioned_deps.expr.data == .e_object) {
```

Comment:
@Jarred-Sumner This is the only line I added in this PR.
The git diff looks noisy due to whitespace changes to indent the `if` block.

Reply:
on ignoring whitespace 
```
git diff HEAD~1 -w
diff --git a/src/bun.js/WebKit b/src/bun.js/WebKit
index 4d6ee410..4010c6c6 160000
--- a/src/bun.js/WebKit
+++ b/src/bun.js/WebKit
@@ -1 +1 @@
-Subproject commit 4d6ee41032866e63aa43ba007616c632ed1c90a4
+Subproject commit 4010c6c6c6ebd3517d4c6d4dfb30a6dd2ec127ac
diff --git a/src/install/npm.zig b/src/install/npm.zig
index 796a88c7..8ea89b20 100644
--- a/src/install/npm.zig
+++ b/src/install/npm.zig
@@ -1191,6 +1191,7 @@ pub const PackageManifest = struct {
 
                     inline for (dependency_groups) |pair| {
                         if (prop.value.?.asProperty(comptime pair.prop)) |versioned_deps| {
+                            if (versioned_deps.expr.data == .e_object) {
                                 const items = versioned_deps.expr.data.e_object.properties.slice();
                                 var count = items.len;
 
@@ -1359,6 +1360,7 @@ pub const PackageManifest = struct {
                                 }
                             }
                         }
+                    }
 
                     if (!parsed_version.version.tag.hasPre()) {
                         release_versions[0] = parsed_version.version;
```